src/peft/tuners/lokr/model.py

```diff
@@ -91,7 +92,7 @@ class LoKrModel(LycorisTuner):
 
     def _create_and_replace(
         self,
-        config: LycorisConfig,
+        config: LycorisConfig | LoKrConfig,
```

Comment:
`LoKrConfig` is a subclass of `LycorisConfig`, so this should not be necessary. Do you see issues caused by this type annotation?

Reply:
Added just for better type annotation; but yeah since `LoKrConfig` is a subclass of `LycorisConfig`, this change would not be needed.